### PR TITLE
feat: claude integration

### PR DESCRIPTION
**Pull request by danorlando**
﻿Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change

- [X] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?
- [X] Conversations with all models with title generation
- [X] Editing settings, saving, editing, exporting and recalling presets 
- [X] Aborting requests
- [X] Tested formatting of code generation, lists, etc.
- Note: reverse proxy is implemented, but has not been tested

### **Test Configuration**:
You'll need to set an `ANTHROPIC_API_KEY` in the .env. (or leave set to user_provided)

Note that documentation has not yet been updated to reflect the adding of this feature.

### CODE REVIEW

**Comment by kuangxiaoye** [comment]
When will Claude's work be merged into the main branch?(✧∀✧)

File: client/public/assets/microsoft.png
Comment (danny-avila) [inline client/public/assets/microsoft.png:1]: look for SVGs, i think we can find some for both of these new assets

**Comment by danny-avila** [comment]
> When will Claude's work be merged into the main branch?(✧∀✧)

it's nearing completion now

File: client/public/assets/microsoft.png
Comment (fuegovic) [reply client/public/assets/microsoft.png:1]: I couldn't find the claude svg so I made it
and I was able to find the microsoft one
[svg assets.zip](https://github.com/danny-avila/LibreChat/files/12024140/svg.assets.zip)

Comment (fuegovic) [reply client/public/assets/microsoft.png:1]: and here's the PaLM logo in svg

**Review by danny-avila — APPROVED** [review]

**Pull request merged**